.gitignore

```diff
@@ -17,3 +17,4 @@ tests/.coverage*
 build/
 tests/report/
 tests/screenshots/
+./idea
```

Comment:
I don't think this should be added see comment at top.

Reply:
Sure, removed